server/models/functions.sql

```diff
@@ -22,11 +22,9 @@ $$
   Converts a binary uuid (16 bytes) to dash-delimited hex UUID (36 characters).
 */
 CREATE FUNCTION BUID(b BINARY(16))
-RETURNS CHAR(36) DETERMINISTIC
+RETURNS CHAR(32) DETERMINISTIC
```

Comment:
Is it necessary to write a migrate script for the API/ change in function? I'm under the impression the backup system doesn't include functions or stored procedures so it may not be needed, just making sure there's a plan for applying this PR to existing systems.